custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -56,6 +56,18 @@ class CloudIssuePage(
         return this;
     }
 
+    override fun linkIssueButton(): AbstractIssuePage {
+        TODO("Not yet implemented")
```

Comment:
`editProbability = 0.10f` in `ScenarioSimilarities`, so VU will try to edit in both Cloud and Server and currently, each edit also does issue linking, so it will start generating errors for Cloud.
And `TODO` throws `NotImplementedError`, which is not an `Exception`, so it will not be handled gracefully, but it will abruptly kill the VU.

We can log "TODO: unimplemented" warnings.
Alternatively, we can assign a probability of linking and set it to zero for both DC and Cloud or at least for Cloud.